packages/keystone/tasks.js

```diff
@@ -173,6 +173,7 @@ function registerTasks (modulesList) {
             Object.values(module).forEach(
                 (task) => {
                     if (task._type !== TASK_TYPE) {
+                        console.error(task)
```

Comment:
debug

Reply:
it's useful when you're adding tasks, it'll work only on local env